fendermint/app/src/cmd/objects.rs

```diff
@@ -98,8 +98,8 @@ cmd! {
                 .and(warp::header::optional::<String>("Range"))
                 .and(warp::query::<HeightQuery>())
                 .and(with_client(client.clone()))
-                .and(with_iroh(iroh_manager.clone()))
-                .and_then(handle_object_download_with_manager);
+                .and(with_iroh(iroh_node.clone()))
```

Comment:
isn't this the "staging" / objects Iroh node? seems like it should be the "chain" / resolver Iroh node, ie, from the notion doc:

> The data is only expected to be available in the iroh-r node, so downloading happens from there.

Reply:
Yes, you are totally right, that was an oversight on my part. This means we need to include the RPC details of the `iroh-r` node in here, which I am not sure on how to go about. Is there any existing connection to the `resolver` parts from this part?

Reply:
there isn't. right now, the only connection to the `ipc` / `fendermint` process is the cometbft rpc endpoint (for reading info about the chain). is there someway we could "proxy" an iroh request to another node?

Reply:
another option: expose a new HTTP endpoint on the manager that we can redirect the HTTP request to from the object API for download:
- new HTTP endpoint only available on local network (for the object API only)
- object API performs the object get request from the chain as it does today (to check it's status)
- then object API forwards the request to this new endpoint

Reply:
>  is there someway we could "proxy" an iroh request to another node?

yes we can, that was my idea, I just need to have the rpc address the same way I do from the syscall setup

Reply:
i see, nice. so somehow we have to make `ENV_IROH_RPC_ADDR` and `ENV_IROH_RPC_KEY` available to the object API. 

here https://github.com/recallnet/ipc/blob/feat-iroh-33/recall/iroh_manager/src/manager.rs#L29, guessing we'll have to bind to all interfaces, and use a predefined port, such that the objects container can reach it.